ui/restmodel/src/main/scala/pl/touk/nussknacker/restmodel/processdetails.scala

```diff
@@ -77,7 +78,10 @@ object processdetails extends JavaTimeEncoders with JavaTimeDecoders {
   sealed trait ProcessShapeFetchStrategy[ProcessShape]
 
   object ProcessShapeFetchStrategy {
-    implicit case object Fetch extends ProcessShapeFetchStrategy[DisplayableProcess]
+    // TODO: Find places where FetchDisplayable is used and think about replacing it with FetchCanonical. We generally should use displayable representation
```

Comment:
This idea is still up to date. I would like to have kind of generic graph (like Displayable) in: database and import/export, but canonization is helpful in rewriting process, and it is why it is used in disabling of nodes or very similar task (dict value substitution). IMO as far as we have canonical process in db, we should use it in rewriting. When we switch to Displayable, we should add some general purpose rewriter (which maybe will partially do canonization) and which should handle both rewriting tasks (disabling and substitution). I'll will put some TODO in rewriter when it will be done... WDYT about this idea?